src/Symfony/Component/Console/Tests/Input/StringInputTest.php

```diff
@@ -71,6 +71,7 @@ public function getTokenizeData()
             ["--long-option='foo bar''another'", ['--long-option=foo baranother'], '->tokenize() parses long options with a value'],
             ["--long-option='foo bar'\"another\"", ['--long-option=foo baranother'], '->tokenize() parses long options with a value'],
             ['foo -a -ffoo --long bar', ['foo', '-a', '-ffoo', '--long', 'bar'], '->tokenize() parses when several arguments and options'],
+            ["--arg='Jenny'\''s'", ["--arg=Jenny's"], '->tokenize() parses quoted quotes'],
```

Comment:
I suggest also adding a case using quoted double quotes, to help avoiding regressions

Reply:
like that?